pipeline_dp/dp_engine.py

```diff
@@ -273,15 +273,13 @@ def sample_unique_elements_fn(pid_and_pks):
 
         return col
 
-    def _drop_partitions(self, col, partitions,
-                         data_extractors: pipeline_dp.DataExtractors):
+    def _drop_partitions(self, col, partitions, partition_extractor: Callable):
```

Comment:
nit: fix the pydoc, public_partitions > partitions.

Reply:
Done